src/pages/home/report/ReportActionCompose/SuggestionMention.tsx

```diff
@@ -281,14 +279,8 @@ function SuggestionMention(
     );
 
     useEffect(() => {
-        if (value.length < previousValue.length) {
-            // A workaround to not show the suggestions list when the user deletes a character before the mention.
-            // It is caused by a buggy behavior of the TextInput on iOS. Should be fixed after migration to Fabric.
-            // See: https://github.com/facebook/react-native/pull/36930#issuecomment-1593028467
-            return;
-        }
```

Comment:
Some notes for this one: 
 - removing this solves the bug when user would delete a character on the mention text and the mentions suggestion list would not appear
 - originally it was introduced here: https://github.com/Expensify/App/pull/29957 (solving issue: https://github.com/Expensify/App/issues/28170)
 - this was described as a bug in `TextInput` but now on this branch `RNMarkdownTextInput` is used and this bug no longer appears
 - I have verified on mac os Chrome, Ios, Android and mWeb ios that without this if we do not experience any kind of mentions flashing when user deletes a character immediately before the `@` char